### ISSUE TITLE
Plug-ins available for Technique H69 under 2.4.1 Bypass Blocks 

### ISSUE THREAD

**Issue by raebened** [w3c/wcag#1146]
Are there standard browser shortcuts or working, updated plug-ins (other than screen readers) that provide header or ARIA navigation assistance?  Sorry if I'm missing something here but I couldn't find an easy solution for sighted keyboard-only users. If there isn't a standard method or generally available plug-in this should not be an acceptable technique.  Please advise.  Thanks.

**Comment by JAWS-test** [comment]
I am not aware that headings or landmarks can be used for keyboard navigation without a screen reader, see also: https://lists.w3.org/Archives/Public/w3c-wai-ig/2011JulSep/0177.html

**Comment by scottaohara** [comment]
There is this browser plugin: http://matatk.agrip.org.uk/landmarks/

**Comment by alastc** [comment]
Given that [H69](https://www.w3.org/TR/WCAG20-TECHS/H69.html) is a sufficient technique for two SCs, I think the only change would be to make it advisory for Bypass blocks, assuming the working group agreed.

**Comment by fstrr** [comment]
This issue looks like it can be closed. If it needs to be re-opened, please do that and convert it to a Discussion.
